MathlibTest/linear_combination.lean

```diff
@@ -384,6 +384,10 @@ h2 : b ≥ 0
 #guard_msgs in
 example (a b : ℚ) (h1 : a ≤ 1) (h2 : b ≥ 0) : a ≤ b := by linear_combination h1
 
+/-- error: coefficients of inequalities in 'linear_combination' must be nonnegative -/
```

Comment:
So the previous error message was "this constant has no effect on the linear combination; it can be dropped from the term"? That sounds confusing indeed.

Reply:
Previously there had been an ugly failure (from the tactic producing a term which didn't solve the goal), together with the warning "this constant has no effect on the linear combination; it can be dropped from the term".  This is better!